### ISSUE TITLE
unicorn for cygwin

### ISSUE THREAD

**Issue by guard-wait** [unicorn-engine/unicorn#1927]
When I use pip install unicorn on Cygwin platform,I found this issue.
![image](https://github.com/unicorn-engine/unicorn/assets/98081165/c69603f3-edd3-4e57-bf14-c75e0a8f8410)
I then manually installed config-host.h and copied it from unicorr-2.0.1.post1 /src/msvc to src/qemu/include/qemu, and the issues seemed to be resolved.But a new problem has emerged.
![image](https://github.com/unicorn-engine/unicorn/assets/98081165/c79b83cb-6457-4e07-a21c-f581a44ab2fb)
So what should I do,sir?I installed it on both Windows and Linux with no problem.But I want to use gef for gdb on windows,so I must install it on cygwin.

**Comment by wtdcode** [comment]
Personally I recommend you using WSL2.

For cygwin we lack testing for a long time and its priority is not very high because few people uses it.

**Comment by ForkInABlender** [comment]
It is supposed to work even for cygwin,

Where is compile documentation?????

**Comment by wtdcode** [comment]
The building steps are the same with native Linux in general.
________________________________

**Comment by ForkInABlender** [comment]
right, but how do we compile it on native linux so it runs for cygwin?

As it does not compile do to missing cmake rules?...

**Comment by ForkInABlender** [comment]
should one have it target the mingw rule for cmake?
